app/src/main/java/io/github/phantasmdragon/buttonx/presentation/view/TimeCountView.kt

```diff
@@ -0,0 +1,259 @@
+package io.github.phantasmdragon.buttonx.presentation.view
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.os.CountDownTimer
+import android.os.Parcel
+import android.os.Parcelable
+import android.text.*
+import android.text.style.AbsoluteSizeSpan
+import android.text.style.TextAppearanceSpan
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.View
+import io.github.phantasmdragon.buttonx.R
+import java.util.*
+
+class TimeCountView @JvmOverloads constructor(
```

Comment:
I know that's the first iteration, but there are loads stuff to change. Merged because not into develop.
Hope see improvements.